pandas/io/pytables.py

```diff
@@ -2484,7 +2484,7 @@ def f(values, freq=None, tz=None):
             return f
         elif klass == PeriodIndex:
             def f(values, freq=None, tz=None):
-                return PeriodIndex._simple_new(values, None, freq=freq)
+                return PeriodIndex._from_ordinals(values, freq=freq)
```

Comment:
We discussed this briefly but never came to a conclusion.  Were you not on board with the idea of making _simple_new the lowest-level constructor for cross-class consistency?

Reply:
Avoid private method